.github/workflows/rust-publish.yml

```diff
@@ -5,12 +5,20 @@ on:
     secrets:
       CARGO_REGISTRY_TOKEN:
         required: true
+    additional-deb-packages:
+      description: 'Space separated list of .deb packages that will be installed on ubuntu.'
+      required: false
+      default: ''
+      type: string
 
 jobs:
 
   publish:
     runs-on: ubuntu-latest
     steps:
+    - name: Install additional dependencies (Linux)
+      if: inputs.additional-deb-packages != ''
+      run: sudo apt-get update && sudo apt-get -y install ${{ inputs.additional-deb-packages }}
     - uses: actions/checkout@v3
     - uses: stellar/actions/rust-cache@main
     - run: rustup update
```

Comment:
@fnando I merged this PR, but I think it would also be fair to add `--no-verify` to the publish which will mean the crate won't be built and you shouldn't need any deps installed.

Adding `--no-verify` might normally be a bit weird, except that all rust repos also use the publish dry run job prior, so we already have a guard against publishing things that don't verify (build).